File: Java Food Order App/Menu.java
package FoodOrderApp;

public class Menu {
	byte menuID;
	String menuName;
	int price;
	String description;
	byte quantity;
	
	static String restaurantName = "Wow Foods";
	
	Menu(byte id, String name, int charge) {
		menuID = id;
		menuName = name;
		price = charge;
	}
	
	static Menu[] getSnacksMenuCard(Menu[] menu) {		
		menu[0] = new Menu((byte)1, "Sandwich", 100);
		menu[1] = new Menu((byte)2, "Pizza", 200);
		menu[2] = new Menu((byte)3, "Burger", 150);
		menu[3] = new Menu((byte)4, "Fries", 90);
		menu[4] = new Menu((byte)5, "Coffee", 50);
		menu[5] = new Menu((byte)6, "Tea", 30);
		return menu;
	}
	
	static Menu[] getMealMenuCard(Menu[] menu) {		
		menu[0] = new Menu((byte)1, "Biryani", 300);
		menu[1] = new Menu((byte)2, "Fried Rice", 250);
		menu[2] = new Menu((byte)3, "Noodles", 220);
		menu[3] = new Menu((byte)4, "Soup", 120);
		menu[4] = new Menu((byte)5, "Dal Khichdi", 150);
		menu[5] = new Menu((byte)6, "Thali", 120);
		return menu;
	}
	

	
	static void displayMenu(Menu[] menu) {
		System.out.println("SrNo\tItemName\tPrice");
		for(int i = 0; i < 6; i++) {
			if(menu[i].menuName == "Sandwich") {
				System.out.println(menu[i].menuID + "\t" + menu[i].menuName + "\t" + (byte)menu[i].price);			
			} else {
				System.out.println(menu[i].menuID + "\t" + menu[i].menuName + "\t\t" + (byte)menu[i].price);
			}
		}
		
	}
}

File: Kotlin Food Order App/Menu.kt
package foodOrderApp

class Menu(menuID: Byte, menuName: String, menuPrice: Short) {
    var menuID: Byte = 0
    var menuName: String = ""
    var menuPrice: Short = 0
    init {
        this.menuID = menuID
        this.menuName = menuName
        this.menuPrice = menuPrice
    }

    companion object {
        fun getSnacksMenuCard(): ArrayList<Menu> {
            val menuArray = arrayListOf<Menu>()
            menuArray.add(Menu(1.toByte(), "Sandwich", 100.toShort()))
            menuArray.add(Menu(2.toByte(), "Pizza", 200.toShort()))
            menuArray.add(Menu(3.toByte(), "Burger", 150.toShort()))
            menuArray.add(Menu(4.toByte(), "Fries", 90.toShort()))
            menuArray.add(Menu(5.toByte(), "Coffee", 50.toShort()))
            menuArray.add(Menu(6.toByte(), "Tea  ", 30.toShort()))
            return menuArray
        }

        fun getMealMenuCard(): ArrayList<Menu> {
            val menuArray = arrayListOf<Menu>()
            menuArray.add(Menu(1.toByte(), "Biryani", 300.toShort()))
            menuArray.add(Menu(2.toByte(), "Fried Rice", 250.toShort()))
            menuArray.add(Menu(3.toByte(), "Noodles", 220.toShort()))
            menuArray.add(Menu(4.toByte(), "Soup", 120.toShort()))
            menuArray.add(Menu(5.toByte(), "Dal Khichdi", 150.toShort()))
            menuArray.add(Menu(6.toByte(), "Thali", 100.toShort()))
            return menuArray
        }

        fun displayMenu(arr: ArrayList<Menu>) {
            println("SrNo\tItemName\t\tPrice")
            for(i in 0.rangeTo(arr.size - 1)) {
                if(arr[i].menuName == "Sandwich") println("${arr[i].menuID}\t\t${arr[i].menuName}\t\t${arr[i].menuPrice}")
                else println("${arr[i].menuID}\t\t${arr[i].menuName}\t\t${arr[i].menuPrice}")
            }
        }
    }
}